.github/workflows/ci.yml

```diff
@@ -89,13 +89,6 @@ jobs:
         run: |
           hatch run coverage run setup.py test
           hatch run coverage xml -i
-      - uses: codecov/codecov-action@v4
-        env:
-          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
-        with:
-          files: coverage.xml
-          fail_ci_if_error: true
-          verbose: true
       - name: Archive Coverage
```

Comment:
Will we ever look at this? Maybe worth deleting, plus the xml step above (but maybe keep the normal coverage run)